resources/measures/hvac_nighttime_oa_controls/measure.rb

```diff
@@ -0,0 +1,206 @@
+        is_unitary_system = true
+      end
+    end
+    return is_unitary_system
+  end
+
+  # determine if the air loop is residential (checks to see if there is outdoor air system object)
+  def air_loop_res?(air_loop_hvac)
+    is_res_system = true
+    air_loop_hvac.supplyComponents.each do |component|
+      obj_type = component.iddObjectType.valueName.to_s
+      case obj_type
+      when 'OS_AirLoopHVAC_OutdoorAirSystem'
+        is_res_system = false
+      end
+    end
+    return is_res_system
+  end
+
+  # Determine if the system is a DOAS based on
+  # whether there is 100% OA in heating and cooling sizing.
+  def air_loop_doas?(air_loop_hvac)
+    is_doas = false
+    sizing_system = air_loop_hvac.sizingSystem
+    if sizing_system.allOutdoorAirinCooling && sizing_system.allOutdoorAirinHeating && (air_loop_res?(air_loop_hvac) == false) && (air_loop_hvac.name.to_s.include?("DOAS") || air_loop_hvac.name.to_s.include?("doas"))
+      is_doas = true
+    end
+    return is_doas
+  end
+
+  # Determine if is evaporative cooler
+  def air_loop_evaporative_cooler?(air_loop_hvac)
+    is_evap = false
+    air_loop_hvac.supplyComponents.each do |component|
+      obj_type = component.iddObjectType.valueName.to_s
+      case obj_type
+      when 'OS_EvaporativeCooler_Direct_ResearchSpecial', 'OS_EvaporativeCooler_Indirect_ResearchSpecial', 'OS_EvaporativeFluidCooler_SingleSpeed', 'OS_EvaporativeFluidCooler_TwoSpeed'
+        is_evap = true
+      end
+    end
+    return is_evap
+  end
+
+  # define what happens when the measure is run
+  def run(model, runner, user_arguments)
+    super(model, runner, user_arguments)  # Do **NOT** remove this line
+
+    # use the built-in error checking
+    if !runner.validateUserArguments(arguments(model), user_arguments)
+      return false
+    end
+
+    # assign the user inputs to variables
+    #space_name = runner.getStringArgumentValue('space_name', user_arguments)
+
+    # check the space_name for reasonableness
+    # if space_name.empty?
+      # runner.registerError('Empty space name was entered.')
+      # return false
+    # end
+
+    # report initial condition of model
+    # runner.registerInitialCondition("The building started with #{model.getSpaces.size} spaces.")
+
+    # # add a new space to the model
+    # new_space = OpenStudio::Model::Space.new(model)
+    # new_space.setName(space_name)
+
+    # # echo the new space's name back to the user
+    # runner.registerInfo("Space #{new_space.name} was added.")
+
+    # # report final condition of model
+    # runner.registerFinalCondition("The building finished with #{model.getSpaces.size} spaces.")
+	
+	template = 'ComStock 90.1-2013'
+    std = Standard.build(template)
+
+    # loop through air loops
+    unitary_system_count = 0
+    li_unitary_systems = []
+    non_unitary_system_count = 0
+    li_non_unitary_systems = []
```

Comment:
What happens to the non unitary systems? Currently, this list is not used for anything.

Reply:
Sorry, I may have made an addition to this locally without pushing it. I do use that list to iterate thru and change the OA schedule, but I was missing the fan schedule change for non-unitary, as you mentioned. : ) I'll push these changes.